tests/data/example_create_labware_messages.py

```diff
@@ -24,7 +24,7 @@
                 "countryOfOrigin": "United Kingdom",
                 "genomeSize": "14",
```

Comment:
What is the unit of this? Is it storage as gigabytes required? I am asking because I have seen `{"genomeSize": "1,234,567,890 bp"}` elsewhere as base pairs.

Reply:
> What is the unit of this? Is it storage as gigabytes required? I am asking because I have seen `{"genomeSize": "1,234,567,890 bp"}` elsewhere as base pairs.

Genome size is stored as a string.